meeting_minutes/2024/2024-06-26.md

```diff
@@ -0,0 +1,131 @@
+  - Use organizations out there to spread and encouraging adoptions.  Who’s using the standard – use names on website for recognition and awareness. 
+  - Simple as a list or open to suggestions on how to provide that information.
+  - Covered by CSAF aggregators? Is that their purpose.  They collect who they are publishing, CSAF may not be compliant, but can’t retrieve automatically.  
+  - Value to have a list but who would collect and maintain.  Include in CSAF 2.1 field to announce it.
+  - Can collect names and find way to convert it over. Promote data quality throughout system. 
+  - Discuss more at next meeting. 
+- Miscellaneous comments:
+  - Sonny van Lingen (Huawei) 6/26/2024 1:18 PM • Thanks for this recap :-)
+  - Thomas Schmidt 6/26/2024 1:25 PM • oasis-csaf@connectedcommunity.org
+  - Vivek Nair (MSFT) 6/26/2024 1:52 PM • Have we looked at the new CNA rules (https://www.cve.org/Media/News/item/blog/2024/05/07/CNA-Rules-v4-0-Update-and-Transition) and its impact on CSAF? 
+
+    I'll raise it on GitHub for discussion next time - If CVE's are to be assigned for Cloud/AI/ML regardless of the need for action by the customer, we might need to distinguish between CVE's requiring action vs those which do not (like, maybe by a flag?).
+  - Sonny van Lingen (Huawei) 6/26/2024 1:58 PM • And at least theoretically one could run a CSAF Trusted Provider with list_on_csaf_aggregators "false" and mirror_on_csaf_aggregators "false" :-)
+  - Martin Prpic (Red Hat) 6/26/2024 1:58 PM • Could just start collecting vendor names in that issue and then convert it into a list for the website.
+  - Thomas Schmidt 6/26/2024 1:59 PM • CSAF lister: wid.cert-bund.de/.well-known/csaf-aggregator/aggregator.json
+  - CSAF aggregator: aggregator.cert-bund.de/.well-known/csaf-aggregator/aggregator.json
+
+
+- The TC discussed the need for mandatory tests to enforce rules on product version ranges in order to write correct details with version ranges.
+- There was a discussion on whether to include EPSS scores in CSAF documents, with some participants supporting the inclusion and others suggesting it should be referenced externally.
+- The issue of allowing sharing groups in the distribution section of CSAF advisories was raised, with the aim of better identifying who receives the advisories.
+- The meeting discussed issues with the scalability of adding individual revision history entries for Red Hat CSAF documents.
+- There was a suggestion to provide guidance in the standard for what the revision history should contain and how to implement automated revision histories.
+- The idea of creating a list of organizations that have adopted the CSAF standard was proposed, with potential methods for collecting and displaying this information discussed.
+
+
+
+## Adjourn
+
+- The meeting was adjourned @ 18:058 UTC
+
+**Note**: All monthly meetings take place on the last Wednesday of each month at 18:00 UTC (19:00 CET, 13:00 EST, 10:00 PST).
```

Comment:
As the time is always the same (except for the European/US daylight saving shift weeks), I think we are meeting according to ET...
```suggestion
**Note**: All monthly meetings take place on the last Wednesday of each month at 13:00 ET (19:00 CEST/19:00 CET, 10:00 PT, 17:00/18:00 UTC).